### ISSUE TITLE
Linux ARM64 release

### ISSUE THREAD

**Issue by trueToastedCode** [git-ecosystem/git-credential-manager#1470]
### Feature description

I seen Linux ARM64 is not covered. Please guys it's 2023 😭

**Comment by dscho** [comment]
@trueToastedCode yes, it's 2023, so everybody knows by now that you contribute the changes to open source software that you want to see, right?

Here is a ticket that should help you with getting started: https://github.com/git-ecosystem/git-credential-manager/issues/606.

As that ticket is a better starting point, I will close this here ticket.
